src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -122,6 +121,8 @@ export default function CDDetails() {
     const pipelines = result[1]['value'].pipelines
     const deploymentAppType = pipelines?.find((pipeline) => pipeline.id === Number(pipelineId))?.deploymentAppType
     const cdPipelinesMap = mapByKey(pipelines, 'environmentId')
+    const _deploymentAppDeleteRequest = mapByKey(pipelines, 'deploymentAppDeleteRequest')
```

Comment:
This will return a map by keys true and false, as deploymentAppDeleteRequest is a boolean. is it expected?

Reply:
Removed